docs/advanced-usage.rst

```diff
@@ -0,0 +1,940 @@
+**************
+Advanced Usage
+**************
+
+This section has examples of using the Python Driver for more advanced use
+cases such as escrow.
+
+.. todo:: Work in progress. Will gradually appear as
+
+    * https://github.com/bigchaindb/bigchaindb/issues/664
+    * https://github.com/bigchaindb/bigchaindb-driver/issues/108
+    * https://github.com/bigchaindb/bigchaindb-driver/issues/109
+    * https://github.com/bigchaindb/bigchaindb-driver/issues/110
+    * https://github.com/bigchaindb/bigchaindb-driver/issues/111
+
+    are taken care of.
+
+
+Getting Started
+===============
+
+First, make sure you have RethinkDB and BigchainDB `installed and running`,
+i.e. you `installed them <https://docs.bigchaindb.com/projects/server/en/latest/dev-and-test/setup-run-node.html>`_ and you ran:
+
+.. code-block:: bash
+
+    $ rethinkdb
+    $ bigchaindb configure
+    $ bigchaindb start
+
+Don't shut them down! In a new terminal, open a Python shell:
+
+.. code-block:: bash
+
+    $ python
+
+Now we can import the :class:`~bigchaindb_driver.BigchainDB` class and create
+an instance:
+
+.. code-block:: python
+
+    from bigchaindb_driver import BigchainDB
+    bdb = BigchainDB()
+
+This instantiates an object ``bdb`` of class
+:class:`~bigchaindb_driver.BigchainDB`. When instantiating a
+:class:`~bigchaindb_driver.BigchainDB` object without arguments (as above), it
+sets the server node URL to ``http://localhost:9984/api/v1``.
+
+
+Create a Digital Asset
+======================
+At a high level, a "digital asset" is something which can be represented
+digitally and can be assigned to a user. In BigchainDB, users are identified by
+their public key, and the data payload in a digital asset is represented using
+a generic `Python dict <https://docs.python.org/3.4/tutorial/datastructures.html#dictionaries>`_.
+
+In BigchainDB, only the federation nodes are allowed to create digital assets,
+by doing a special kind of transaction: a ``CREATE`` transaction.
+
+.. code-block:: python
+
+    from bigchaindb_driver.crypto import generate_keypair
+
+    # Create a test user: alice
+    alice = generate_keypair()
+
+    # Define a digital asset data payload
+    digital_asset_payload = {'msg': 'Hello BigchainDB!'}
+
+    # All transactions need to be signed by the user creating the transaction.
+    # Write the transaction to the bigchain.
+    # The transaction will be stored in a backlog where it will be validated,
+    # included in a block, and written to the bigchain.
+    tx = bdb.transactions.create(verifying_key=alice.verifying_key,
+                                 signing_key=alice.signing_key,
+                                 payload=digital_asset_payload)
```

Comment:
This has changed now, as shown in `usage.rst`: it should be `asset=`. Maybe just put a big WARNING box at the top of this page that says the whole thing is more or less a work-in-progress, here be dragons, etc.